evaluate/repository.go

```diff
@@ -60,6 +60,9 @@ func Repository(logger *log.Logger, resultPath string, model evalmodel.Model, la
 
 			continue
 		}
+		if assessments[metrics.AssessmentKeyProcessingTime] == 0 {
+			return nil, nil, pkgerrors.Errorf("no model response time measurement present for %q at repository %q", model.ID(), repositoryPath)
```

Comment:
Shouldn't this be handled the same way the `err` response from `model.GenerateTestsForFile` is handled because as soon as one file does not work with the model it will break the whole execution instead of marking that particular file as "errored"

Reply:
Hmmm. Well... if there is no response time measurement it's definitely our fault not the models. So that means we forgot to implement the timing measurement for the model. And I think that justifies a hard error.

Reply:
Just to ensure we can never forget to implement the timing when we add new models.